src/components/auth/LoginForm.vue

```diff
@@ -27,12 +32,16 @@ const developmentEnvironment = process.env.NODE_ENV === "development";
         </h2>
     </div>
     <div class="mt-10 sm:mx-auto sm:w-full sm:max-w-[480px] flex justify-center">
-        <div class="bg-verse-200/10 dark:bg-verse-500/20 backdrop-blur-sm px-6 py-12 shadow-2xl rounded-lg sm:px-12 w-11/12">
+        <div
+            class="bg-verse-200/10 overflow-hidden dark:bg-verse-500/20 backdrop-blur-sm px-6 py-12 shadow-2xl rounded-lg sm:px-12 w-11/12">
             <div v-if="isLoggedIn">
                 <div class="text-center flex flex-col gap-8 text-verse-900 dark:text-verse-100 w-full">
                     <span>
                         You are logged in as <a class="underline" href="/user/me"> {{ user?.full_name }}</a>
                     </span>
+                    <div v-if="willRedirect">
+                        redirecting in {{ Math.round(countdown / 1000) }}s
```

Comment:
Suggestion for another time: when it reaches `0`, we could show the label `Redirecting...` instead of `Redirecting in 0s`

Reply:
i think i fixed it :3